src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -150,7 +150,7 @@
         :disabled="!!error || verifying"
         :icon="verifying ? AnimatedSpinner : null"
         :text="verifying ? $t('common.verifying') : $t('common.confirm')"
-        @click="popupProps?.resolve()"
+        @click="confirm"
```

Comment:
```suggestion
        @click="confirm()"
```
To avoid passing the `$event` as the 1st argument.